skyportal/enum_types.py

```diff
@@ -20,7 +48,9 @@ def force_render_enum_markdown(values):
     cfg.get('spectrum_types.types', ['source', 'host', 'host_center'])
 )
 ALLOWED_MAGSYSTEMS = tuple(val['name'] for val in _MAGSYSTEMS.get_loaders_metadata())
-ALLOWED_BANDPASSES = tuple(val['name'] for val in _BANDPASSES.get_loaders_metadata())
+# though in the registry, the additional bandpass names are not in the _BANDPASSES list
```

Comment:
Hrm, should be in `sncosmo.bandpasses._BANDPASSES`.

Either way, should probably use a `set` below, to avoid the duplication you guard for above.

Reply:
done.